src/app/modules/ix-forms/components/ix-slide-in/components/ix-modal-header/ix-modal-header.component.ts

```diff
@@ -11,5 +12,13 @@ export class IxModalHeaderComponent {
   @Input() loading: boolean;
   @Input() disableClose = false;
 
-  constructor(public slideInService: IxSlideInService) {}
+  constructor(
+    public slideInService: IxSlideInService,
+    private slideIn2Service: IxSlideIn2Service,
+  ) {}
+
+  close(): void {
+    this.slideInService.close();
+    this.slideIn2Service.closeEvent$.next(true);
```

Comment:
I don't think you should call something called `closeEvent` directly. You should probably `closeLast` here.